data-pipeline/events-router/src/main/config/events-router.properties

```diff
@@ -27,7 +27,7 @@ job.container.count=__events_router_yarn_container_count__
 
 # Task
 task.class=org.ekstep.ep.samza.task.EventsRouterTask
-task.inputs=kafka.__env__.telemetry.denorm.valid
+task.inputs=kafka.__env__.telemetry.denorm.valid,kafka.__env__.telemetry.log
```

Comment:
Log events should also be validated. Send the stream via druid validator.

Reply:
done